pkgs/applications/audio/uhhyou.lv2/default.nix

```
@@ -0,0 +1,77 @@
+{ stdenv
+, fetchgit
+, fftw
+, libGL
+, libX11
+, libjack2
+, liblo
+, lv2
+, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  pname = "uhhyou.lv2";
+  version = "unstable-2020-05-27";
+
+  src = fetchgit {
+    url = "https://github.com/ryukau/LV2Plugins";
+    rev = "f20d1c8a624d3a757f820b5c41c70f6f9bb6dc22";
+    deepClone = true;
+    sha256 = "12g9j4rrhkw91sbnqi8xg8wkl5j34m0l7fdwhqa2ny526n52lq7k";
+  };
+
+  buildInputs = [
+    fftw
+    libGL
+    libX11
+    libjack2
+    liblo
+    lv2
+  ];
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  prePatch = ''
+    patchShebangs generate-ttl.sh
+    cp patch/NanoVG.cpp lib/DPF/dgl/src/NanoVG.cpp
+  '';
+
+  enableParallelBuilding = true;
+
+  installPhase = ''
+        mkdir -p $out/lib/lv2/
+        mkdir -p $out/lib/lxvst/
+        mkdir -p $out/bin
+        cd bin
+    ls
```

Make sure this comes out right.

```suggestion
         ls
```

---

Fixed!